sagemaker-python-sdk/tensorflow_script_mode_training_and_serving/tensorflow_script_mode_training_and_serving.ipynb

```diff
@@ -0,0 +1,239 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TensorFlow script mode training and serving\n",
+    "\n",
+    "Script Mode is refering to a new format of TensorFlow training script supported by SageMaker. Where you can execute any generic TensorFlow training script in SageMaker with minimal to zero modification. Our bootstraping library handles transfering your script to the SageMaker training instance. Then it simply sets up some training related enrivonment variables and executes your training script.\n",
```

Comment:
some thoughts:
* Let's refrain from calling it "new" since it's not like we're going to update this notebook in x days/weeks/months once it's not "new."
* "generic TF script" seems like it could be limiting. what counts as generic? shouldn't we just advertise that customers should be able to run almost any script?
* "zero modification" isn't really true, so let's not use that wording
* "our bootstrapping library" sounds weird seeing as someone who's brand new to SageMaker and at this point of the notebook has no idea what the Python SDK is
* spelling: referring, bootstrapping, transferring, environment

How about:

Script Mode is a training script format for TensorFlow that lets you execute any TensorFlow training script in SageMaker with minimal modification. The SageMaker Python SDK handles transferring your script to a SageMaker training instance; on the training instance, SageMaker's native TensorFlow support simply sets up some training-related environment variables and executes your training script.

Reply:
actually, rereading my comments from before - since this notebook is demonstrating both training and inference, let's mention that the SageMaker Python SDK helps train and deploy models rather than just mentioning training.